contents/python/introduction-to-python.md

```diff
@@ -12,6 +12,15 @@ For absolute beginners, check out [this video](https://www.youtube.com/watch?v=N
 
 Both newbies and experienced programmers can also benefit from [The Python Tutorial](https://docs.python.org/3/tutorial/index.html), which aims to introduce readers to Python's unique features and style.
 
+## Virtual Environment
+
+When starting new projects or hopping on existing Python repositories, you will be recommended to install dependencies using [virtual environment](https://docs.python.org/3/tutorial/venv.html) to remove the possibility of dependencies conflict. This is a good practice especially when managing dependencies from different projects which relies on different python versions and packages.
+
+The [official python documentation](https://docs.python.org/3/tutorial/venv.html) gives a standard way of creating a virtual environment - defining a directory location and activating it. However, you can consider using other libraries than can make this process smoother. Some of the most popular ones are:
```

Comment:
Python natively supports the creation of virtual environments. 
The official Python documentation doesn't give a way to do so, it only provides a tutorial on how to do so.